cmd/swarm-smoke-pss/main.go

```diff
@@ -0,0 +1,182 @@
+	"github.com/ethereum/go-ethereum/metrics/influxdb"
+
+	"github.com/ethersphere/swarm/log"
+	swarmmetrics "github.com/ethersphere/swarm/metrics"
+	"github.com/ethersphere/swarm/tracing"
+
+	cli "gopkg.in/urfave/cli.v1"
+)
+
+var (
+	gitCommit string // Git SHA1 commit hash of the release (set via linker flags)
+)
+
+var (
+	allhosts          string
+	hosts             []string
+	inputSeed         int
+	wsPort            int
+	verbosity         int
+	timeout           int
+	pssMessageTimeout int
+	pssMessageCount   int
+	pssMessageSize    int
+)
+
+func main() {
+
+	app := cli.NewApp()
+	app.Name = "smoke-test-pss"
+	app.Usage = ""
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:        "hosts",
+			Value:       "",
+			Usage:       "comma-separated list of swarm hosts",
+			Destination: &allhosts,
+		},
+		cli.IntFlag{
+			Name:        "ws-port",
+			Value:       8546,
+			Usage:       "ws port",
+			Destination: &wsPort,
+		},
+		cli.IntFlag{
+			Name:        "seed",
+			Value:       0,
+			Usage:       "input seed in case we need deterministic upload",
+			Destination: &inputSeed,
+		},
+		cli.IntFlag{
+			Name:        "verbosity",
+			Value:       1,
+			Usage:       "verbosity",
+			Destination: &verbosity,
+		},
+		cli.IntFlag{
+			Name:        "timeout",
+			Value:       120,
+			Usage:       "timeout in seconds after which kill the process",
+			Destination: &timeout,
+		},
+		cli.IntFlag{
+			Name:        "msgtimeout",
+			Value:       1,
+			Usage:       "timeout in seconds until a message is received",
+			Destination: &pssMessageTimeout,
+		},
+		cli.IntFlag{
+			Name:        "msgcount",
+			Value:       10,
+			Usage:       "number of pss messages that should be send in the pss smoke test",
+			Destination: &pssMessageCount,
+		},
+		cli.IntFlag{
+			Name:        "msgbytes",
+			Value:       128,
+			Usage:       "size of a randomly generated message. defined in bytes",
+			Destination: &pssMessageSize,
+		},
+	}
+
+	app.Flags = append(app.Flags, []cli.Flag{
+		utils.MetricsEnabledFlag,
+		swarmmetrics.MetricsInfluxDBEndpointFlag,
+		swarmmetrics.MetricsInfluxDBDatabaseFlag,
+		swarmmetrics.MetricsInfluxDBUsernameFlag,
+		swarmmetrics.MetricsInfluxDBPasswordFlag,
+		swarmmetrics.MetricsInfluxDBTagsFlag,
+	}...)
+
+	app.Flags = append(app.Flags, tracing.Flags...)
+
+	app.Commands = []cli.Command{
+		{
+			Name:   "asym",
+			Usage:  "send and receive multiple messages across random nodes using asymmetric encryption",
+			Action: wrapCliCommand("asym", pssAsymCheck),
+		},
+		{
+			Name:   "sym",
+			Usage:  "send and receive multiple messages across random nodes using symmetric encryption",
+			Action: wrapCliCommand("sym", pssSymCheck),
+		},
+		{
+			Name:   "raw",
+			Usage:  "send and receive multiple raw messages across random nodes",
+			Action: wrapCliCommand("raw", pssRawCheck),
+		},
+		{
+			Name:   "all",
+			Usage:  "send and receive raw, sym and asym messages across random nodes",
```

Comment:
But in sequence, not mixed. Perhaps be specific about it.

Reply:
not sure if i understand but i changed the comment anyway